test/org/ojalgo/machine/TestBasicMachine.java

```diff
@@ -0,0 +1,63 @@
+package org.ojalgo.machine;
+
+import org.junit.jupiter.api.Test;
+import org.ojalgo.TestUtils;
+
+public class TestBasicMachine {
```

Comment:
Is it possible to get also tests for users of this class such as `Hardware` or `VirtualMachine`?

Reply:
We had a few tests for these classes, though not as high coverage as BasicMachine, and also with heavy use of reflection. I'll try to re-factor some of these.